baremetalpxeserver/baremetalpxeserver/pxeserveragent.py

```
@@ -154,17 +156,19 @@ def _get_capacity(self):
         return total, total - used
 
     def _start_pxe_server(self):
-        ret, _, err = bash_roe("ps -ef | grep -v 'grep' | grep 'dnsmasq -C {0}' || dnsmasq -C {0} -u root".format(self.DNSMASQ_CONF_PATH))
+        ret, _, err = bash_roe(
```

Ruff (0.8.2)

`bash_roe` may be undefined, or defined from star imports

(F405)